static/js/redux/ui/Semesterly.tsx

```diff
@@ -242,4 +244,19 @@ const Semesterly = () => {
   );
 };
 
-export default Semesterly;
+const SemesterlyWithErrorBoundary = withErrorBoundary(Semesterly, {
+  FallbackComponent: FallBack,
+  onError(error, info) {
+    const { componentStack } = info;
+    const { name, message, stack } = error;
+    const errorInfo = {
+      name,
+      message,
+      stack,
+      componentStack,
+    };
+    reportUIError(errorInfo);
+  },
+});
+
+export default SemesterlyWithErrorBoundary;
```

Comment:
does this new export name have to be reflected/changed anywhere else?

Reply:
yeah for readability purpose, I will change it in `init.jsx` to reflect this new name